examples/tracetest-cli-and-agent-with-tracetest-io/README.md

```diff
@@ -12,4 +12,7 @@ To add Tracetest to your GitHub pipeline and use [Tracetest CLI](https://docs.tr
 
 You can get your Tracetest Token from the Tracetest settings page by following the directions described [here](https://docs.tracetest.io/concepts/environment-tokens). The Tracetest Agent APIKey can be seen on the `Agent` tab on `Settings` page.
 
+If you don't need to start an Agent on your GitHub pipeline, you can declare only the `token` parameter and use it. You can see more details on [Running Trace-based Tests using Tracetest CLI with Tracetest.io
```

Comment:
```suggestion
If you don't need to start an Agent on your GitHub pipeline, you can declare only the `token` parameter and use it. You can see more details on [Running Trace-based Tests using the Tracetest CLI with Tracetest.io.
```

Reply:
I think we have a problem with explaining the options we provide.

It took me 5 mins of reading to understand what is going on.

Maybe we need to explain here that the github action has access to the cli and agent but this sample only uses the cli without agent?

Reply:
It makes sense. I'll create a new PR to improve this description.